e2e/portalicious/pages/RegistrationsPage.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, Locator } from '@playwright/test';
+import { Page } from 'playwright';
+
+import BasePage from './BasePage';
+
+class RegistrationsPage extends BasePage {
+  readonly page: Page;
+  readonly tableRows: Locator;
+  readonly selectAllPasCheckbox: Locator;
```

Comment:
I'm thinking that maybe it would make sense to have a `TableModule` like we have for the tests on the old portal. There is a lot of code now being repeated or spread in several pages, when they all use the same table. You don't necessarily have to do this (at all, or in this PR) but just some food for thought.

Reply:
Thought the same but then I thought it would be an overkill for this test. I will put it on my backlog. We definitely need it cause it will speed up a lot of future tests especially the ones testing very small things eg. fsps, updating names etc.